### DISCUSSION TITLE
Historical Vapor Pressure

### DISCUSSION THREAD

**Discussion by dsheehan312** [open-meteo/open-meteo#1093] [General]
Hi,

Is there by chance a way to pull Vapor Pressure from the historical API, or should I go about calculating it using the existing temperature and humidity data. I see Vapor Pressure deficit, but not vapor pressure itself.

Thanks!

**Comment by patrick-zippenfenig** [comment]
Hi, you have to calculate it yourself. The API only exposes vapor pressure deficit. For which use case do you need vapor pressure?
